### PR TITLE
wmoid.adoc

### PR DESCRIPTION
**Pull request by vturpin**

### CODE REVIEW

**Comment by soerenthomsen** [comment]
Hi @vturpin please give your PR's a clear title and description to make it more clear what has been changed. 
"Update OG_Format.adoc" is not very specific. 

Merci :)

**Review by castelao — COMMENTED** [review]
I remember the discussion on not been mandatory, which I agree. But should it be removed or maintained as desired instead of mandatory?

**Comment by vturpin** [comment]
WMO ID must remain mandatory. 
All GOOS platforms have a WMO ID. 
This is also mandatory to push data on the GTS.
It is really at the base of the OceanGliders monitoring and many data management (GDAC) applications.

**Comment by castelao** [comment]
@vturpin , I'm confused. I think this PR removes the mention of WMO ID. Was that the intention?

**Comment by vturpin** [comment]
I though the issue was about having WMOID as a variable and the same information "wmo id" as a global attribute. Both mandatory. So I've PR to remove the attribute and keep the "WMO ID" as a mandatory variable. 
But maybe I miss understood the issue #22

**Review by justinbuck — APPROVED** [review]
this simply makes WMO mandatory in line with OCG recomendations

**Review by glidermann — APPROVED** [review]
Approved: WMO ID as a variable aligns with our philosophy of reserving global attributes to refer to a grouping of deployments from a number of platforms when possible. Closing issue #22

**Comment by soerenthomsen** [comment]
Looking at what this PR will do it will remove the WMO as mandatory. 
Please clarify before merging!!!

Screenshot 2022-06-03 at 10 35 33

File: OG_Format.adoc [hunk 1]
```diff
@@ -107,7 +107,6 @@ https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/colle
 
  |mandatory |“Autonomous Underwater Vehicle”
 |platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
-|wmoid |Wmo identifier |mandatory |
 |id a|
```
Comment (soerenthomsen) [inline OG_Format.adoc:110]: at the moment it will remove the wmoid to be mandatory, but if I understand you want to keep it mandatory. please clarify!

**Review by soerenthomsen — CHANGES_REQUESTED** [review]

File: OG_Format.adoc (hunk 1 above)
Comment (glidermann) [reply OG_Format.adoc:110]: Mandatory as global attribute is removed, but mandatory as variable remains! We don't need both and are in agreement to keep only as variable.

**Comment by justinbuck** [comment]
> Approved: WMO ID as a variable aligns with our philosophy of reserving global attributes to refer to a grouping of deployments from a number of platforms when possible. Closing issue #22

@glidermann The concept of using the WMO to merge deployments in the same way does not carry form Argo (where all deployments have a unique WMO vs glider where the same WMO has multiple deployments). I agree the MO should be mandatory, but for merging of deployments we need to use the trajectory ID attribute, as per IOOS.

**Review by justinbuck — APPROVED** [review]
I agree WMO should be madatory.

**Review by callumrollo — APPROVED** [review]

**Review by soerenthomsen — APPROVED** [review]

**Pull request merged**